src/Cohesity.Powershell/Scripts/Vlan/Remove-CohesityVlan.ps1

```diff
@@ -0,0 +1,56 @@
+function Remove-CohesityVlan {
+    <#
+        .SYNOPSIS
+        Removes a vlan.
+        .DESCRIPTION
+        The Remove-CohesityVlan function is used to delete vlan.
+        .NOTES
+        Published by Cohesity
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Remove-CohesityVlan  -VlanId 18
```

Comment:
Can't we do this by name ? Or can you make sure we can do piping ?

Reply:
VlanId is a key in creating a new vlan.  Cohesity UI accepts a unique VlanId from the user. Therefore mimicking the same behaviour. Piping has been introduced for remove operation.